docs/Users/Tutorials/Git_Help.rst

```diff
@@ -230,6 +220,27 @@ fine by our reviewers, you can simply send it again like this. If not, edit it
 and send it.
 Now you have successfully edited your last commit!
 
+Keeping your fork in sync
+-------------------------
+
+As people work on coala new commits will be added. This will result in your
+local fork going out of sync with the remote repository.
+To sync your changes with the remote repository run the following commands:
+
+::
+
+    $ git checkout master
+    $ git git pull origin master
```

Comment:
Also, please explain this whole process. A newcomer will not probably have a remote named `upstream`.